common/lib/xblock/xblock_discussion/xblock_discussion.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+"""
+Discussion XBlock
+"""
+import json
+import logging
+
+from xblockutils.resources import ResourceLoader
+from xblockutils.studio_editable import StudioEditableXBlockMixin
+
+from xblock.core import XBlock
+from xblock.fields import Scope, String, UNIQUE_ID
+from xblock.fragment import Fragment
+
+log = logging.getLogger(__name__)
+loader = ResourceLoader(__name__)  # pylint: disable=invalid-name
+
+
+@XBlock.needs('user')
+class DiscussionXBlock(XBlock, StudioEditableXBlockMixin):
+    """ Provides functionality similar to discussion XModule in inline mode """
```

Comment:
This docstring should say "Provides a discussion forum that is inline with other content in the courseware" or something. It doesn't help to say "similar to discussion XModule" since we're also deleting that at the same time.